mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java

```diff
@@ -138,6 +139,9 @@ public void onClick(DialogInterface dialog, int which) {
                                 PrefManager.clearPrefs();
                                 startActivity(new Intent(MifosBaseActivity.this,
                                         SplashScreenActivity.class));
+                                Toast.makeText(MifosBaseActivity
```

Comment:
Since **Toaster** is used instead of **Toast** throughout the application, it is expected to use **Toaster** and not **Toast** here as well. You can find it under **com.mifos.mifosxdroid.core.util** package.

Reply:
I think you shouldn't use Toaster as it's switching between fragments. And Toaster is a snackbar.